tests/models/test_serialized_dag.py

```diff
@@ -120,5 +150,5 @@ def test_bulk_sync_to_db(self):
         dags = [
             DAG("dag_1"), DAG("dag_2"), DAG("dag_3"),
         ]
-        with assert_queries_count(7):
+        with assert_queries_count(10):
```

Comment:
Will optimize the above queries to lower this count in followup PRs

Reply:
Oh this is added an extra query for each (bulk) write?

Reply:
Yeah, one to check if DAG is updated or not.